### PR TITLE
Updating static IPs used by Cypress Cloud

### PR DESCRIPTION
**Pull request by jmherbst**
Updating static IPs used by Cypress Cloud

### CODE REVIEW

File: docs/faq/questions/cloud-faq.mdx [hunk 1]
```diff
@@ -261,5 +261,8 @@ Cloud.
 you may also need to add both the following to the version control IP allow
```
Comment (MikeMcC399) [inline docs/faq/questions/cloud-faq.mdx:261]: "both" is no longer correct if there are no longer just 2 IP addresses to consider.
```suggestion
you may also need to add the following to the version control IP allow
```

Comment (jmherbst) [reply docs/faq/questions/cloud-faq.mdx]: Good catch, Thank you Mike

File: docs/faq/questions/cloud-faq.mdx [hunk 2]
```diff
@@ -258,8 +258,11 @@ Cloud.
...
 **If you are using GitHub Enterprise or GitLab for Enterprise (Self-managed),
-you may also need to add both the following to the version control IP allow
+you may also need to add the following to the version control IP allow
 list:**
 
+- `35.169.145.173` - **Dedicated IP**
+- `44.199.152.70` - **Dedicated IP**
+- `52.70.95.89` - **Dedicated IP**
 - `3.211.102.119` - **Dedicated IP**
 - `18.213.72.78` - **Dedicated IP**
```
Comment (MikeMcC399) [inline docs/faq/questions/cloud-faq.mdx:268]: Sort
```suggestion
- `3.211.102.119` - **Dedicated IP**
- `18.213.72.78` - **Dedicated IP**
- `35.169.145.173` - **Dedicated IP**
- `44.199.152.70` - **Dedicated IP**
- `52.70.95.89` - **Dedicated IP**
```

**Review by MikeMcC399 — APPROVED** [review]
LGTM
although you might want to update the text in the original post, since no entries are marked as deprecated and there are no dates anymore.

**Review by MikeMcC399 — APPROVED** [review]

**Review by jaffrepaul — APPROVED** [review]

**Pull request merged**